tests/test_match.py

```diff
@@ -140,3 +158,26 @@ def test_padding(self):
         res = gmp_adopted.fit(G1, G2)
 
         assert 1.0 == (sum(res.perm_inds_ == np.arange(n)) / n)
+
+    def test_sim(self):
+        n = 150
```

Comment:
is there a sensible test where this is smaller? Been trying to streamline the tests a lot

Reply:
i can lower n to 75 maybe? GM on 150 nodes is pretty quick though

Reply:
@bdpedigo thoughts?

Reply:
sure 75 seems fine?